common/defs/README.md

```diff
@@ -86,7 +86,7 @@ If you want to add a **wallet link**, modify the file [`wallets.json`](wallets.j
 # Support Information
 
 We keep track of support status of each built-in coin over our devices. That is
-`trezor1` for Trezor One, `trezor2` for Trezor T, `connect` for [Connect](https://github.com/trezor/connect)
+`t1b1` for Trezor One, `t2t1` for Trezor T, `t2b1` for Trezor R, `connect` for [Connect](https://github.com/trezor/connect)
```

Comment:
please change to upper-case everywhere where it's externally visible (so dict keys in support.json, argument names, etc. Keeping it lowercase in local variables is ok)

I also think that aesthetically it would be nice if we called it in lowercase originally :) but it's not -- `T2B1` is a fixed four-byte ASCII case-sensitive sequence and we should treat it as such.

Reply:
Renamed all in [b684b89](https://github.com/trezor/trezor-firmware/pull/3240/commits/b684b89d94bd0f3f4fe713e9a7397c1f69550cb7)